src/controllers/midi/midimessage.h

```diff
@@ -8,6 +8,7 @@
 #include <cstdint>
 
 #include "preferences/usersettings.h"
+#include "util/qhash.h"
```

Comment:
I think we have a `compatibility.h` header file that we should use instead of adding a new header for each Qt6 class. Or maybe add a single new `qtcompat.h` file?

Reply:
A single file would pull in many unused Qt dependencies. We already have `qtmutex.h`.

We should definitely agree on some pattern. Introducing a new subdirectory `util/qtcompat` with separate header files could also be an option.